src/main/groovy/io/saagie/plugin/dataops/models/PipelineOverride.groovy

```diff
@@ -1,4 +1,4 @@
 package io.saagie.plugin.dataops.models
 
-class PipelineOverride extends PropertyOverride{
+class PipelineOverride extends ArtifactPropertyOverride{
```

Comment:
same remark as above

Reply:
Same response "we structered like this thinking that there will be contain in those classes but it didn't happen, i will remove them"